plugin/frontend/page.php

```diff
@@ -258,7 +258,7 @@ function images( $client, $dir, $options, $skip, $remaining ) {
 			$description = $file->getDescription();
 			$val = [
 				'id'        => $file->getId(),
-				'description' => ($description === NULL ? '' : esc_attr($description)),
+				'description' => (isset($description) ? '' : esc_attr($description)),
```

Comment:
Now the expressions need to be the other way around though :)

Reply:
:D